sale_stock_usability/sale_stock.py

```diff
@@ -2,11 +2,43 @@
 # @author Alexis de Lattre <alexis.delattre@akretion.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields
+from odoo import models, fields, api
 
 
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
     warehouse_id = fields.Many2one(track_visibility='onchange')
     incoterm = fields.Many2one(track_visibility='onchange')
+    picking_status = fields.Selection([
+        ('deliverd', 'Fully deliverd'),
+        ('partialy delivered', 'Partialy Delivered'),
+        ('to deliver', 'To Deliver'),
```

Comment:
I would prefer to avoid spaces in the keys of a selection field. I know that Odoo SA did that on a field on sale.order, but they are not an example to follow...

Reply:
Ok

Reply:
I made the changes